charts/pega/charts/hazelcast/templates/clustering-service-migration.yaml

```diff
@@ -0,0 +1,137 @@
+{{ if and (and (eq (include "performDeployment" .) "true") (eq (include "isMigrationInit" .) "true"))
+   (eq (include "isClusteringServiceEnabled" .) "true") }}
+---
```

Comment:
A client deployment that is migrated in a certain upgrade. Should they turn the `migration.enabled` to false in the subsequent upgrades? If so, is it conveyed in the documentation

Reply:
@kishorv10 , Updated in readme